agent/config/config.go

```diff
@@ -54,13 +54,33 @@ const (
 	// DefaultDockerStopTimeout specifies the value for container stop timeout duration
 	DefaultDockerStopTimeout = 30 * time.Second
 
+	// DefaultImageCleanupTimeInterval specifies the default value for image cleanup duration. It is used to
+	// remove the images pulled by agent.
+	DefaultImageCleanupTimeInterval = 30 * time.Minute
+
+	// DefaultNumImagesToDeletePerCycle specifies the default number of images to delete when agent performs
+	// image cleanup.
+	DefaultNumImagesToDeletePerCycle = 5
+
+	//DefaultImageDeletionAge specifies the default time duration that an image
+	// can be deleted since it has been pulled.
```

Comment:
This comment is confusing.  Perhaps something like "specifies the minimum amount of elapsed time after an image has been pulled before it can be deleted."